Makefile

```diff
@@ -21,14 +21,15 @@ all: test
 
 ##@ Build
 .PHONY: build
-build: ## Bazel build
-	bazel build //cmd/cip:cip
+build: ## Go build
+	go build -o cip ${REPO_ROOT}/cmd/cip/main.go
 	${REPO_ROOT}/go_with_version.sh build ${REPO_ROOT}/test-e2e/cip-auditor/cip-auditor-e2e.go
 	${REPO_ROOT}/go_with_version.sh build ${REPO_ROOT}/test-e2e/cip/e2e.go
 
 .PHONY: install
 install: ## Install
-	bazel run //:install-cip -c opt -- $(shell go env GOPATH)/bin
+	make build
```

Comment:
```suggestion
```

The `build` target gets captured in the previous review comment.

Reply:
Ack.